tests/source/expr-overflow-delimited.rs

```diff
@@ -0,0 +1,111 @@
+// rustfmt-overflow_delimited_expr: true
```

Comment:
Could you add a test with a comment before the overflowing argument please?

Reply:
Boop!